### DISCUSSION TITLE
Resized Font

### DISCUSSION THREAD

**Discussion by Quint097** [Quint097/DWIN_Font_Files#3] [Announcements]
- Resized Droid Sans Mono font
- Fixed uneven zero

<hr /><em>This discussion was created from the release <a href='https://github.com/Quint097/DWIN_Font_Files/releases/tag/v1.1'>Resized Font</a>.</em>

**Comment by gothcha** [comment]
Hi, thanks again for the fantastic work, like the new home for fonts 😊

**Comment by Cal-LF** [comment]
@Quint097 Flashed the latest commit successfully.
